### PR TITLE
Deploy a system update: Clarify the IP addr to be used by virtual device

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by oleorhagen** [comment]
Would it not be an idea to use the tabbed UI feature here, similar to [here](https://staging.docs.mender.io/development/downloads)

File: 01.Get-started/03.Deploy-a-system-update/docs.md [hunk 1]
```diff
@@ -67,6 +67,12 @@ Setup the `IP_ADDRESS` shell variable with correct IP address of your device:
 IP_ADDRESS="<DEVICE-IP-ADDRESS>"
 ```
 
+If you are unsure, check the device inventory reported to Mender. Look for
+`ipv4_eth0` attribute or similar.
```
Comment (kacf) [inline 01.Get-started/03.Deploy-a-system-update/docs.md:71]: I think `ipv4_wlan0` is also a pretty common one.

**Review by kacf — APPROVED** [review]

**Comment by lluiscampos** [comment]
@oleorhagen Could you have a quick look on the second commit?

**Review by oleorhagen — APPROVED** [review]
Nice!

**Pull request merged**